pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -114,57 +112,6 @@ type PipelineResourceBinding struct {
 	ResourceSpec *resource.PipelineResourceSpec `json:"resourceSpec,omitempty"`
 }
 
-// PipelineResourceResult used to export the image name and digest as json
-type PipelineResourceResult struct {
```

Comment:
thanks @JeromeJu!

`PipelineResourceResult` is used as a means to read the container status and make it available to the `taskRun` status, for example, `startedAt`,  `exitCode`, `taskResults`, failure message, etc.

This struct introduced `ResultType` to support identifying internal usage v/s pipeline resources. 

Now, since pipeline resources are being deleted, please evaluate to see if `ResourceName` and `ResultType` are needed or not. Thanks!

PS: https://github.com/tektoncd/pipeline/commit/a9e7b54d8ea7e9aee2b5b2367a5588f6abf5e188

Reply:
Thanks @pritidesai , I will track this and bring up the refactoring in followup PRs:)